src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -44,6 +44,10 @@ public static void PrintProcessStatus(IConsole console)
                     {
                         sb.Append($"{process.Id, 10} {process.ProcessName, -10} [Elevated process - cannot determine path]\n");
                     }
+                    catch (NullReferenceException)
```

Comment:
How are we getting a `NullReferenceException` here in the first place?  The docs seem to indicate that we should get the `InvalidOperationException` if we are unable to get `ProcessName`.  Are we running up against something else?  If we aren't and this is indeed an expected result, should we mirror this exception below as well?

Reply:
We're getting it because `process.MainModule` is null, not `ProcessName`.